FRCSoftware2023.csv

```diff
@@ -16,10 +16,9 @@ RadioConfigTool Israel,FRC_Radio_Configuration_23_0_2_IL.zip,https://firstfrc.bl
 2023Manual,2023FRCGameManual.pdf,https://firstfrc.blob.core.windows.net/frc2023/Manual/2023FRCGameManual.pdf,00000000000000000,false
 NavX,navx-mxp.zip,https://www.kauailabs.com/public_files/navx-mxp/navx-mxp.zip,00000000000000000,true
 NavX Library,NavX-Offline-2023-0-3.zip,https://dev.studica.com/maven/release/2023/NavX-Offline-2023-0-3.zip,c2df1e4ad86f778a93d1c7d5facce2ee,true
-REV Hardware Client,REV-Hardware-Client-Setup-1.5.2.exe,https://github.com/REVrobotics/REV-Software-Binaries/releases/download/rhc-1.5.2/REV-Hardware-Client-Setup-1.5.2.exe,26b0d65fad47f565461a54675ee118ce,false
+REV Hardware Client w/ bundled firmware,REV-Hardware-Client-Setup-1.5.2-with-FRC-firmware.zip,https://github.com/REVrobotics/REV-Software-Binaries/releases/download/rhc-1.5.2/REV-Hardware-Client-Setup-1.5.2-with-FRC-firmware.zip,8328fd1045decf42e3d55671df2720aa,true
```

Comment:
probably won't allow the `/` character due to windows pathing

Reply:
Only the friendly name has the `/`. The filename only has dashes.

Reply:
Hopefully not?